src/utils/git.ts

```diff
@@ -22,6 +30,23 @@ export async function getLatestTag(git: SimpleGit): Promise<string> {
   return (await git.raw('describe', '--tags', '--abbrev=0')).trim();
 }
 
+export async function getChangesSince(
+  git: SimpleGit,
+  rev: string
+): Promise<GitChange[]> {
+  const { all: commits } = await git.log({
+    from: rev,
+    to: 'HEAD',
+    symmetric: false,
```

Comment:
I don't really understand this option. Why are we using it here?

https://www.npmjs.com/package/simple-git#git-log

https://git-scm.com/docs/gitrevisions#_dotted_range_notations

Reply:
This option defaults to `true`, giving you a "triple-dot range" which lists all commits from both parts whereas we only need the commits reachable from `HEAD` that don't exist on `rev`.

Makes sense? One is `A - B` and the other is more like `A XOR B`. We want the former.

Reply:
Added some comments: https://github.com/getsentry/craft/pull/289